c2corg_ui/templates/base.html

```diff
@@ -50,7 +50,7 @@
     <button class="btn btn-default menu-open-close header"><span class="glyphicon glyphicon-align-justify"></span></button>
     <div class="page-header">
       <div class="quick-search">
-        <app-simple-search app-loading></app-simple-search>
+        <app-search app-loading app-search-map="" dataset="wro"></app-search>
```

Comment:
As explained previously, the directive is now "app-simple-search" and no longer "app-search".